How_to_guides/Analyse_multiple_polygons.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>We're getting a runtime value here, and a quick search suggests that matplotlib is flagging us trying to cast a NaN to a colour in the colourmapping function (and not handling the error). Do you think that's correct? Is this something we should flag with matplotlib or some difference between odc.geo and datacube.utils in specifying number formats? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
![image](https://github.com/GeoscienceAustralia/dea-notebooks/assets/20038594/4adf1806-1924-4c07-bd3c-24d631d5238d)